lib/EnsEMBL/REST/Model/Overlap.pm

```diff
@@ -345,13 +345,13 @@ sub variation {
 
 sub structural_variation {
   my ($self, $slice) = @_;
-  my @so_terms = $self->_get_SO_terms();
-  my ($source, $include_evidence, $somatic) = (undef)x3;
-  my $sv_class = (@so_terms) ? $so_terms[0] : ();
+#  my @so_terms = $self->_get_SO_terms();
```

Comment:
If you're removing code could you please remove and not comment it out. We can always go back through the history to find it again. Thanks.